people.yaml

```diff
@@ -1833,4 +1849,4 @@ amir-qayyum-arbisoft:
     email: amir.qayyum@arbisoft.com
     name: Amir Qayyum Khan
     agreement: institution
-    institution: MIT
\ No newline at end of file
+    institution: MIT
```

Comment:
not sure why the diff is marking this line as being deleted. If you view the file it's still there